pytest_copie/plugin.py

```diff
@@ -76,9 +76,11 @@ def copy(
                     '"_subdirectory" key is missing from copier.yaml'
                 )
 
+            repo_name = extra_answers.get("repo_name") or params["repo_name"]["default"]
```

Comment:
You know what? Not everyone uses a key called "repo_name", on second thought I don't think it is a good idea to hard code it here

Reply:
agreed see my first comment